src/notifications/reducer.ts

```diff
@@ -0,0 +1,84 @@
+import { createReducer } from 'redux-act'
+
+import * as actions from './actions'
+
+export interface State {
+    readNotificationList: {
+        notifications: any[]
+        resultExhausted: boolean
+    }
+    unreadNotificationList: any[]
+    showMoreIndex?: any
```

Comment:
`showMoreIndex` is the id of notification. It can be also undefined. This is used for `Show More`, so it will not associate with the `NotifDefinition`. :slightly_smiling_face: